components/autofill/src/sync/address/incoming.rs

```diff
@@ -118,22 +129,20 @@ fn save_incoming_tombstones(
     signal: &dyn Interruptee,
 ) -> Result<()> {
     log::info!("staging {} incoming tombstones", incoming_tombstones.len());
-    let chunk_size = 1; // XXX - chunk_size of 1 seems wrong?
     sql_support::each_sized_chunk(
         &incoming_tombstones,
-        sql_support::default_max_variable_number() / chunk_size,
+        sql_support::default_max_variable_number(),
```

Comment:
I thought I wrote this yesterday but apparently not :( I think you can change here to use `sql_support::each_sized` and drop the 2nd param, and `sql_support::repeat_multi_values` can change to `sql_support::repeat_sql_values` which also means you can drop the 2nd param, so that odd `chunk_size` can be removed entirely.